package.json

```diff
@@ -52,5 +52,8 @@
         "typedoc": "^0.22.13",
         "typescript": "^4.6.2",
         "typescript-esm": "^2.0.0"
+    },
+    "resolutions": {
+        "**/@toruslabs/solana-embed/@solana/web3.js": "=1.36.0"
```

Comment:
We can't do this. This will cause the end user developer to end up with two versions of web3.js in their code. In addition to bloat, this will cause all kinds of compile-time and runtime issues (such as type mismatches). Torus should probably update their version of web3.js used so that it's appropriately permissive (i.e., use `^`).

Reply:
@toruslabs/solana-embed is using web3.js version `^1.43.0`
I am having issue to build the wallet-adapter repo as it complaint about two version of web3.js (1.36.0 in adapter and 1.43.0 in torus)
Should torus downgrade the version back to `^1.36.0` ?